robot_skills/src/robot_skills/head.py

```diff
@@ -387,167 +386,6 @@ def clear_face(self):
         rospy.loginfo('clearing all learned faces')
         self._clear_srv()
 
-    def _get_persons(self, image):
```

Comment:
Where did all this stuff go?

Reply:
https://github.com/tue-robotics/people_detector